appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,120 @@
+$location=$args[0]
+$repoProvider=$env:APPVEYOR_REPO_PROVIDER
+$projectSlug=$env:APPVEYOR_PROJECT_SLUG
+$repoName=$env:APPVEYOR_REPO_NAME
+$repoCommit=$env:APPVEYOR_REPO_COMMIT
+#$repoProvider='gitHub'
+#$projectSlug='sarif-sdk'
+##$projectSlug='sarif-sdk-931m5'
+#$repoName='Rolstenhouse/sarif-sdk'
+#$repoCommit='f82ebd892917bd6157efc04b832a57964ec1f2c6'
+
+$githubBaseUri = ""
+
+if($repoProvider -eq "gitHub")
+{
+  $githubBaseUri = "https://raw.githubusercontent.com/"
+}
+
+function Rebase-Uri($originalURI)
+{
+  $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString($originalURI.IndexOf("c:/")))
```

Comment:
Ryley, the file URL will be of the form `file:///c:/path/file`, so he's stripping the leading `file:///`.

As for the C: drive, I noticed that myself, but I decided to let it pass because we "know" what the AppVeyor's VMs look like. But I wouldn't object to making this a little more resilient by using `"file:///".Length"` as the starting index.